docs/src/pages/components/switches/UnstyledSwitchesMaterial.js

```diff
@@ -1,436 +0,0 @@
-/* eslint-disable no-restricted-imports, react/prop-types */
```

Comment:
It's not ok to disable `no-restricted-imports` file-wide
1. for the `TouchRipple` specifically since that's an internal component
2. in general for `no-restricted-imports` since that hides potential problems with every import that's added in the future